podcasts/View+CVPixelBuffer.swift

```diff
@@ -37,10 +43,11 @@ extension View {
         }
 
         let renderer = ImageRenderer(content: self)
+        renderer.isOpaque = true
         renderer.scale = scale
-        renderer.render { size, render in
-            render(context)
-        }
+
+        let cgImage = renderer.cgImage!
+        context.draw(cgImage, in: CGRect(x: 0, y: 0, width: size.width, height: size.height))
```

Comment:
This ended up being faster than rendering to the context + actually respected the `scale` above.